superset-frontend/src/SqlLab/components/AceEditorWrapper.tsx

```diff
@@ -22,14 +22,14 @@ import 'brace/mode/sql';
 import 'brace/theme/github';
 import 'brace/ext/language_tools';
 import ace from 'brace';
-import { areArraysShallowEqual } from '../../reduxUtils';
-import sqlKeywords from '../utils/sqlKeywords';
+import { areArraysShallowEqual } from 'src/reduxUtils';
```

Comment:
is this from a lint rule? auto modifications from your IDE? Could we do a programmatic migration here to absolute paths everywhere?

Reply:
We probably could do that, but I'm not sure it's worth it. I think I made a decision to change it here because it made more sense than a local path. But sometimes local paths are nice.